demo/src/main/scala/com/tresata/spark/scalding/demo/DemoJobSequenceClass.scala

```diff
@@ -0,0 +1,23 @@
+package com.tresata.spark.scalding.demo
```

Comment:
should we add these extra classes to demo? i checked them in by accident, but it doesnt hurt i guess